### ISSUE TITLE
Admin user no longer has admin access

### ISSUE THREAD

**Issue by eporter23** [emory-libraries/dlp-selfdeposit#178]
After our deploys 3/26 - 3/27, the default Admin user no longer has admin access to the Hyrax arch instance.
Note: Slack indicates there was a deploy on 3/27,  but the version history in the footer still shows 3/26.

To replicate:
Login as admin
View the Dashboard
Note that you cannot see all the extended Settings and other Admin dashboard features

Also, when trying to access /roles, the application returns a "you are not authorized to view this page"

**Comment by bwatson78** [comment]
When we installed `hydra-role-management`, we circumvented what should have been a development-only setting that allowed 3 users (1 admin, 2 not) to be automatically created. The admin user was appointed a fake admin Role, but that goes away as soon as the system detects that the Role Management is being used.

I have created a proper Admin Role instance and assigned it to `admin@example.com`. I have confirmed that that user can now access Roles and other Admin options.
